docs/how-to-build.md

```diff
@@ -4,244 +4,232 @@ slug: /how-to-build
-  :::
+:::info
+- Gravitino skips all Docker-related tests by default.
 
-3. Get the Gravitino server binary package.
+  To run Docker-related tests, make sure you have installed Docker, and
+  (1) set `skipDockerTests=false` in the `gradle.properties` file, or
+  (2) use `-PskipDockerTests=false` in the command line, or
+  (3) `export SKIP_DOCKER_TESTS=false` in the Shell.
 
-   ```shell
-   ./gradlew compileDistribution
-   ```
+<!--TODO(Qiming): move the following two items elsewhere-->
+- macOS uses [docker-connector](https://github.com/wenjunxiao/mac-docker-connector)
+  to make the Gravitino Trino connector work. Refer to
+  `$GRAVITINO_HOME/dev/docker/tools/mac-docker-connector.sh`, and
+  `$GRAVITINO_HOME/dev/docker/tools/README.md` for more details.
+  The Gravitino Trino connector uses JDK17 to build.
+  For example, to avoid vendor-related issues on macOS,
+  Gravitino uses the Amazon Corretto OpenJDK 17 to build the Trino connector.
 
-  The `compileDistribution` command creates a `distribution` directory in the Gravitino root directory.
+- You can use [OrbStack](https://orbstack.dev) as an alternative for Docker for macOS.
+  With OrbStack, you can run Gravitino integration tests without `docker-connector`.
+:::
 
-  :::note
-  The `./gradlew clean` command deletes the `distribution` directory.
-  :::
 
-4. Assemble the Gravitino server distribution package.
+1. Add the apt repository for Docker-CE:
 
    ```shell
-   ./gradlew assembleDistribution
+   curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+   sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
    ```
 
-  The `assembleDistribution` command creates `gravitino-{version}-bin.tar.gz` and `gravitino-{version}-bin.tar.gz.sha256` under the `distribution` directory.
-
-  You can deploy these to your production environment.
-
-  :::note
-  The `gravitino-{version}-bin.tar.gz` file is the Gravitino **server** distribution package, and the `gravitino-{version}-bin.tar.gz.sha256` file is the sha256 checksum file for the Gravitino server distribution package.
-  :::
-
-5. Assemble the Gravitino Trino connector package
+1. Install the `docker-ce` package and start the Docker daemon:
 
-  ```shell
-   ./gradlew assembleTrinoConnector
+   ```shell
+   sudo apt update
+   sudo apt install docker-ce
+   sudo service docker start
    ```
 
-  or
+1. Verify the installation by running `hello-world`:
 
    ```shell
-   ./gradlew assembleDistribution
+   sudo docker run hello-world
    ```
 
-  This creates `gravitino-trino-connector-{version}.tar.gz` and
-  `gravitino-trino-connector-{version}.tar.gz.sha256` under the `distribution` directory. You can uncompress and deploy it to Trino to use the Gravitino Trino connector.
-
-6. Assemble the Gravitino Iceberg REST server package
+1. Add yourself to the `docker` group so that you can run Docker commands without `sudo` in the future.
```

Comment:
" In the future, you should add yourself to the `docker` group to run Docker commands without `sudo`."

Reply:
No. That is not what I meant.
I meant "adding yourself to the group NOW", and then "you don't need sudo IN THE FUTURE".
Makes sense?

Reply:
In that case, just say "You should add yourself to the docker group to run Docker commands without sudo."